### PR TITLE
Add basic Dockerfile

### PR DESCRIPTION
**Pull request by gvso**
This is a simple Dockerfile for running the tool in CLI mode. 

To deal with Google Cloud credentials, mounting `gcloud`'s default credentials should be enough
```
gcloud auth application-default login
docker run -v=$HOME/.config/gcloud:/root/.config/gcloud release-operator -project=myproject -interval=600
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.14
```
Comment (ahmetb) [inline Dockerfile:1]: use multi stage dockerfiles pls

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+RUN mkdir /src
```
Comment (ahmetb) [inline Dockerfile:3]: use WORKDIR

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+COPY . /src
+WORKDIR /src/cmd/operator
+
+RUN go build -mod=readonly -o operator
```
Comment (ahmetb) [inline Dockerfile:7]: add some static compilation flags so it can work without glibc

File: Dockerfile (hunk 1 above)
Comment (gvso) [reply Dockerfile:1]: Hmm, distroless images. That's interesting 😃

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+FROM golang:1.14 AS builder
+WORKDIR /src
+COPY . .
+RUN CGO_ENABLED=0 GOOS=linux go build -mod=readonly -v -o operator ./cmd/operator
```
Comment (ahmetb) [inline Dockerfile:4]: `-o /bin/operator`

good name for binary: `cloud_run_release_operator`

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+FROM gcr.io/distroless/static
+WORKDIR /root
```
Comment (ahmetb) [inline Dockerfile:7]: you can just do this `COPY --from=builder /bin/operator /bin/operator`
ENTRYPOINT ["/bin/operator", "-cli"]

File: Dockerfile [hunk 6]
```diff
@@ -0,0 +1,8 @@
+FROM golang:1.14 AS builder
+WORKDIR /src
+COPY . .
```
Comment (ahmetb) [inline Dockerfile:3]: to speed up builds (at least locally on dev machine) you can do:

```
COPY go.mod go.sum ./
RUN go mod download (+ whatever readonly flag goes here if any)
COPY . . 
[...]
```

**Pull request merged**